### PR TITLE
Use 'emailNameFrom' when sending email (and use 'emailFrom' as fallback)

### LINKED ISSUE #931: "Organization Email Name" setting is not honoured
I tried sending an email, and it worked.

However the "Organization Email Name" setting had no effect.

### PR DESCRIPTION
**Pull request by ikusteu**
Fixes #931 

This needs to be deployed first (the updated `sendEmail`) and checked that way...I wrote a test verifying that the expected payload is handed off for delivery, but can't verify for sure that it will work e2e as expected (until and actual mail has beed sent, that is...).

### CODE REVIEW

File: packages/functions/src/sendEmail/https.ts [hunk 1]
```diff
@@ -90,7 +91,7 @@ export const sendEmail = functions
 
       // Construct an email for process delivery
       const email = {
-        from: emailFrom,
+        from: emailNameFrom || emailFrom,
```
Comment (silviot) [inline packages/functions/src/sendEmail/https.ts:94]: I think we should combine name and email here in the form of a "mailbox": https://superuser.com/questions/1625686/what-is-the-email-address-format-name-email-called

File: packages/client/src/__tests__/sendEmail.test.ts [hunk 2]
```diff
@@ -159,4 +163,51 @@ describe("SendEmail", () =>
...
+    // Check the process document
+    const deliveryDocPath = deliveryDocumentPath;
+    const deliveryDoc = await adminDb.doc(deliveryDocPath).get();
+    const deliveryDocData = deliveryDoc.data();
+
+    expect(deliveryDocData).toEqual(
+      expect.objectContaining({
+        payload: expect.objectContaining({
+          // Using email name from instead of email from
+          from: "Eisbuk Team",
```
Comment (silviot) [inline packages/client/src/__tests__/sendEmail.test.ts:206]: That's not a vali email address

**Review by silviot — CHANGES_REQUESTED** [review]

File: packages/functions/src/sendEmail/https.ts (hunk 1 above)
Comment (ikusteu) [reply packages/functions/src/sendEmail/https.ts:94]: Oook...had the same idea, but decided to cut it as I wasn't sure if it was my overcomplicating things (plus I didn't know the standard format, so thanks for this...)

File: packages/client/src/__tests__/sendEmail.test.ts (hunk 2 above)
Comment (ikusteu) [reply packages/client/src/__tests__/sendEmail.test.ts:206]: I assume the previous comment fix will address this too, right?

**Review by silviot — APPROVED** [review]

File: packages/client/src/__tests__/sendEmail.test.ts (hunk 2 above)
Comment (silviot) [reply packages/client/src/__tests__/sendEmail.test.ts]: Yeah. I wanted to add that for clarity

**Pull request merged**